source/integrations/generate-lets-encrypt-certificate-using-certbot-for-minio.md

```diff
@@ -0,0 +1,72 @@
+# Generate Let's Encrypt certificate using Certbot for MinIO [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)
+[Let’s Encrypt](https://letsencrypt.org/) is a new free, automated, and open source, Certificate Authority.
+
+[Certbot](https://certbot.eff.org/) is a console based certificate generation tool for Let's Encrypt.
+
+In this recipe, we will generate a Let's Encypt certificate using Certbot. This certificate will then be deployed for use in the MinIO server.
+
+## 1. Prerequisites
+- Install MinIO Server from [here](https://docs.min.io/docs/minio-quickstart-guide).
```

Comment:
Can we do :ref: links in .md files?
Otherwise, https://min.io/docs/minio/linux/reference/minio-server/minio-server.html